test-data/unit/fine-grained-suggest.test

```diff
@@ -175,6 +175,19 @@ Foo('lol')
 (str) -> None
 ==
 
+[case testSuggestTryText]
+# flags: --py2
+# suggest: --try-text foo.foo
+[file foo.py]
+def foo(s):
+    return s
+[file bar.py]
+from foo import foo
+foo('lol')
+[out]
+(unicode) -> unicode
```

Comment:
Huh, why is this not Text?

Reply:
I think because the real underlying typing is called `unicode`, not `Text`. `pyannotate` will print it as `Text`, though, so it is fine in practice.